Installation/cmake/modules/CGAL_Macros.cmake

```diff
@@ -411,11 +411,20 @@ if( NOT CGAL_MACROS_FILE_INCLUDED )
 
   macro( create_CGALconfig_files )
 
+    include(CMakePackageConfigHelpers)
+
     # CGALConfig.cmake is platform specific so it is generated and stored in the binary folder.
     configure_file("${CGAL_MODULES_DIR}/CGALConfig_binary.cmake.in"  "${CMAKE_BINARY_DIR}/CGALConfig.cmake"        @ONLY)
+    write_basic_package_version_file("${CMAKE_BINARY_DIR}/CGALConfigVersion.cmake"
+      VERSION "${CGAL_MAJOR_VERSION}.${CGAL_MINOR_VERSION}.${CGAL_BUILD_VERSION}"
+      COMPATIBILITY AnyNewerVersion)
 
     # There is also a version of CGALConfig.cmake that is prepared in case CGAL in installed in CMAKE_INSTALL_PREFIX.
     configure_file("${CGAL_MODULES_DIR}/CGALConfig_install.cmake.in" "${CMAKE_BINARY_DIR}/config/CGALConfig.cmake" @ONLY)
+    configure_file(
+      "${CMAKE_BINARY_DIR}/CGALConfigVersion.cmake"
+      "${CMAKE_BINARY_DIR}/config/CGALConfigVersion.cmake"
+      COPYONLY)
```

Comment:
The `configure_file()` only copies the package version file generated in `${CMAKE_BINARY_DIR}` into the `${CMAKE_BINARY_DIR}/config` folder.

This is necessary if the `CGALConfig.cmake` in the `config` folder could be used directly from the build tree to satisfy a `find_package()` call.

If that's not the case (i.e. this file should only be used after install and from the installation directory), then this `configure_file()` call can be removed and the `INSTALL(FILES)` can be changed to use the version file from the `${CMAKE_BINARY_DIR}` just like you've said.